src/content/translations/id/developers/docs/transactions/index.md

```diff
@@ -0,0 +1,187 @@
+
+Klien Ethereum seperti Geth akan menangani proses penandatanganan ini.
+
+Contoh pemanggilan [JSON-RPC](https://eth.wiki/json-rpc/API):
+
+```json
+{
+  "id": 2,
+  "jsonrpc": "2.0",
+  "method": "account_signTransaction",
+  "params": [
+    {
+      "from": "0x1923f626bb8dc025849e00f99c25fe2b2f7fb0db",
+      "gas": "0x55555",
+      "maxFeePerGas": "0x1234",
+      "maxPriorityFeePerGas": "0x1234",
+      "input": "0xabcd",
+      "nonce": "0x0",
+      "to": "0x07a565b7ed7d7a678680a4c162885bedbb695fe0",
+      "value": "0x1234"
+    }
+  ]
+}
+```
+
+Contoh tanggapan:
+
+```json
+{
+  "jsonrpc": "2.0",
+  "id": 2,
+  "result": {
+    "raw": "0xf88380018203339407a565b7ed7d7a678680a4c162885bedbb695fe080a44401a6e4000000000000000000000000000000000000000000000000000000000000001226a0223a7c9bcf5531c99be5ea7082183816eb20cfe0bbc322e97cc5c7f71ab8b20ea02aadee6b34b45bb15bc42d9c09de4a6754e7000908da72d48cc7704971491663",
+    "tx": {
+      "nonce": "0x0",
+      "maxFeePerGas": "0x1234",
+      "maxPriorityFeePerGas": "0x1234",
+      "gas": "0x55555",
+      "to": "0x07a565b7ed7d7a678680a4c162885bedbb695fe0",
+      "value": "0x1234",
+      "input": "0xabcd",
+      "v": "0x26",
+      "r": "0x223a7c9bcf5531c99be5ea7082183816eb20cfe0bbc322e97cc5c7f71ab8b20e",
+      "s": "0x2aadee6b34b45bb15bc42d9c09de4a6754e7000908da72d48cc7704971491663",
+      "hash": "0xeba2df809e7a612a0a0d444ccfa5c839624bdc00dd29e3340d46df3870f8a30e"
+    }
+  }
+}
+```
+
+- `mentah` adalah transaksi yang ditandatangani dalam bentuk Prefiks Panjang Rekursif (RLP) yang dikodekan
+- `tx` adalah transaksi yang ditandatangani dalam bentuk JSON
+
+Dengan hash tanda tangan, transaksi bisa dibuktikan secara kriptografi berasal dari pengirim dan dikirim ke jaringan.
+
+## Jenis transaksi {#types-of-transactions}
+
+Di Ethereum ada beberapa jenis transaksi yang berbeda:
+
+- Transaksi reguler: transaksi dari satu dompet ke dompet lainnya.
+- Transaksi penerapan kontrak: transaksi tanpa alamat 'kepada', di mana bidang data digunakan untuk kode kontrak.
+
+### Tentang gas {#on-gas}
+
+Seperti yang disebutkan, transaksi memerlukan [gas](/developers/docs/gas/) untuk dijalankan. Transaksi transfer sederhana membutuhkan 21.000 unit Gas.
+
+Jadi agar Bob dapat mengirimkan kepada Alice 1 ETH dengan `baseFeePerGas` sebesar 190 gwei dan `maxPriorityFeePerGas` sebesar 10 gwei, Bob harus membayar biaya berikut:
+
+```
+(190 + 10) * 21000 = 4,200,000 gwei
+--or--
+0.0042 ETH
+```
+
+Akun Bob akan didebit **-1,0042 ETH**
+
+Akun Alice akan dikredit **+1,0 ETH**
+
+Biaya pokoknya akan terbakar **-0,00399 ETH**
+
+Penambang menerima tips **+0,000210 ETH**
+
+Gas juga dibutuhkan untuk interaksi kontrak pintar mana pun.
+
+![Diagram menunjukkan cara mengembalikan dana gas yang tidak terpakai](../../../../../developers/docs/transactions/gas-tx.png) _Diagram diadaptasi dari [Ethereum EVM yang diilustrasikan](https://takenobu-hs.github.io/downloads/ethereum_evm_illustrated.pdf)_
```

Comment:
This image is broken in deploy preview but it is correct - (same path as [Chinese docs](https://ethereum.org/zh/developers/docs/transactions/))